gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/DatastoreTest.java

```diff
@@ -691,6 +696,37 @@ public void testQueryPaginationWithLimit() throws DatastoreException {
   }
 
   @Test
+  public void testEventuallyConsistentQuery() {
+    DatastoreRpcFactory rpcFactoryMock = EasyMock.createStrictMock(DatastoreRpcFactory.class);
+    DatastoreRpc rpcMock = EasyMock.createStrictMock(DatastoreRpc.class);
+    EasyMock.expect(rpcFactoryMock.create(EasyMock.anyObject(DatastoreOptions.class)))
+        .andReturn(rpcMock);
```

Comment:
Maybe we can first create the `DatastoreOptions` and use that object instead of `EasyMock.anyObject(DatastoreOptions.class)`